hnn_core/tests/test_pick_connection.py

```diff
@@ -0,0 +1,236 @@
+# Authors: Nick Tolley <nicholas_tolley@brown.edu>
+#          George Dang <george_dang@brown.edu>
+
+import os.path as op
+import pytest
+
+import hnn_core
+from hnn_core import read_params, Network
+from hnn_core.network import pick_connection
+
+hnn_core_root = op.dirname(hnn_core.__file__)
+
+
+@pytest.fixture(scope="module")
+def base_network():
```

Comment:
haven't we defined this elsewhere too?

https://github.com/jonescompneurolab/hnn-core/blob/d28ddb75fb9c2f12c5fda100efbfdf1c66261d9c/hnn_core/network_models.py#L312

or

https://github.com/jonescompneurolab/hnn-core/blob/d28ddb75fb9c2f12c5fda100efbfdf1c66261d9c/hnn_core/tests/conftest.py#L75

just want to be careful of fragmenting the codebase with too many functions which do similar things